src/types/Ctype.ts

```diff
@@ -1,10 +1,17 @@
 import * as sdk from '@kiltprotocol/sdk-js'
+import ICTypeMetadata, { IMetadata } from '@kiltprotocol/sdk-js/build/types/CTypeMetedata'
 
-export interface ICType {
+export interface ICType extends sdk.ICTypeMetadata {
   cType: sdk.ICType
-  metaData: {
-    author: string
-  }
+}
+
+export interface CTypeMetadataChain {
+  cType: sdk.CType
+  metaData: ICTypeMetadata
+}
+export interface CTypeMetadata {
```

Comment:
This reads, as if this only contains ctype metadata and not cype AND metadata

Reply:
renamed this interface to `CTypeWithMetadata`